src/docquery/pipeline.py

```diff
@@ -66,9 +76,10 @@ def get_pipeline(checkpoint=None, revision=None, device=None):
     else:
         model = checkpoint
 
-    pipeline_kwargs = {}
     if config.model_type == "vision-encoder-decoder":
         pipeline_kwargs["feature_extractor"] = model
+    elif config.model_type == "layoutlm-tc":
+        pipeline_kwargs["max_answer_len"] = 1000  # Let the token classifier filter things out
```

Comment:
nit: Is there any way to have this depend on an attribute from the `config` instead of it being hard-coded?

Reply:
Sorry I forgot to reply to this. I don't think so, or at least it would be "un-kosher" since it's not a property of the model (but rather the post-processing/pipeline code). I think we could incorporate it inside the pipeline though, based on which model it is. The upstreamed version of the pipeline (in transformers) already does that for a few different types of models.

Reply:
Yeah, it would seem more sensible to accept `None` here and have that treated optionally in `decode_spans` (https://github.com/impira/docquery/blob/7a862d91be63ad56e6f6627395c3cdbcb423b405/src/docquery/ext/qa_helpers.py#L62). The kind of change I had in mind, FWIW, is something like:
```
# Disable truncating the answer in `decode_spans` by setting `max_answer_len` to a value
# greater than the maximum number of tokens.
pipeline_kwargs["max_answer_len"] = tokenizer.model_max_length + 1
```

Reply:
Will update